modules/custom/az_finder/src/Plugin/views/exposed_form/QuickstartExposedFilters.php

```diff
@@ -78,8 +79,11 @@ public function exposedFormAlter(&$form, FormStateInterface $form_state): void {
               'js-active-filter-count',
               'ml-1',
```

Comment:
If others agree, could we add a little more to the left margin to set the number apart from the text?

```suggestion
              'ml-2',
```